src/Domain/CampoKeys.php

```diff
@@ -5,21 +5,21 @@
 namespace Fau\DegreeProgram\Common\Domain;
 
 /**
- * @psalm-type CampoKeysMap = array<value-of<self::SUPPORTED_CAMPO_KEYS>, string>
+ * @psalm-type CampoKeysMap = array<value-of<self::SUPPORTED_CAMPO_KEYS>, array<int, string>|string>
```

Comment:
It is a bit confusing that campo key parts might be array or string. Can we use array constantly? Maybe this way `CampoKeys::fromHisCode` can accept several hisCodes: `fromHisCode(string ...$hisCode)`.

Reply:
Thanks for the suggestion. As [discussed](https://inpsyde.slack.com/archives/C045KFJ9MB5/p1725442000141659) in Slack, the fromHisCode method has been removed, and the existing code has been refactored.